apps/stirling-pdf/0.36.1/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   stirling-pdf:
-    image: frooodle/s-pdf:0.36.1
+    image: frooodle/s-pdf:0.36.2
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
The only difference between these two lines is the version number of the Docker image used:

From:
```yaml
image: frooodle/s-pdf:0.36.1
```

To:
```yaml
image: frooodle/s-pdf:0.36.2
```

This change should be safe to implement since 0.36.2 likely includes bug fixes and improvements over 0.36.1. However, ensure that there's no compatibility issue with other settings in the service configuration.